annotator-core/src/main/java/edu/ucr/cs/riple/core/metadata/index/Error.java

```diff
@@ -37,25 +42,26 @@ public class Error extends Enclosed {
   public final String messageType;
   /** Error message. */
   public final String message;
-  /**
-   * If non-null, this error involved a pseudo-assignment of a @Nullable expression into a @NonNull
-   * target, and this field is the Symbol for that target.
-   */
-  @Nullable public final Location nonnullTarget;
-
+  /** The fixes which can resolve this error (possibly empty). */
+  private final Set<Fix> resolvingFixes;
```

Comment:
Can this be an `ImmutableSet<Fix>`?

Reply:
5ac48c160e4b3d6e319c014f922c54ad74234527